detox/provision/plugins.ts

```
@@ -285,8 +285,15 @@ export async function installRequiredPlugin(
 
     if (isInactive) {
         logInfo(`Plugin ${pluginId} is installed but inactive, enabling...`);
-        const enableRes = await client.request<ApiErrorBody>('POST', `/api/v4/plugins/${encodeURIComponent(pluginId)}/enable`, {}, token);
-        if (enableRes.status >= 400) {
+        // ponytail: .catch — a transient enable timeout (e.g. demo-plugin on a
```

ESLint

[error] Expected line before comment.

(lines-around-comment)